veles/secrets/perplexity/detector.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package perplexityapikey contains a Veles Secret type and a Detector for
+// Perplexity API keys (prefix `pplx-`).
+package perplexityapikey
```

Comment:
It looks like our internal usage of Go doesn't support packages that have different names than their containing directory.

Can you rename the directory from perplexity to perplexityapikey?